openscm/core.py

```diff
@@ -256,19 +275,27 @@ def get_writable_timeseries_view(
         parameter = self._get_or_create_parameter(
             name, self._get_or_create_region(region)
         )
-        timeseries = Timeseries(start_time, period_length)
-        parameter.attempt_write(unit, ParameterType.TIMESERIES, timeseries)
-        return WritableTimeseriesView(parameter, unit, timeseries)
+        parameter.attempt_write(unit, timeseries_type, time_points)
+        return WritableTimeseriesView(
+            parameter,
+            unit,
+            time_points,
+            timeseries_type,
+            interpolation_type,
+            extrapolation_type,
+        )
 
-    def get_parameter_info(self, name: Tuple[str], region: Tuple[str]) -> ParameterInfo:
+    def get_parameter_info(
+        self, name: Tuple[str], region_name: Tuple[str]
+    ) -> Optional[ParameterInfo]:
         """
         Get a parameter or ``None`` if not found.
 
         Parameters
         ----------
         name
             :ref:`Hierarchical name <parameter-hierarchy>` of the parameter
-        region
+        region_name
             Hierarchical name of the region or ``()`` for "World".
```

Comment:
```suggestion
            Hierarchical name of the region".
```

I thought we got rid of this?

Reply:
We missed this probably